src/routes/authorRouter.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express');
+const { 
+    createAuthor,
+    getAllAuthors,
+    getAuthorById,
+    updateAuthor, 
+    deleteAuthor,
+} = require('../controllers/author');
+
+const authorRouter = express.Router();
+
+authorRouter.route('/')
+.post(createAuthor)
+.get(getAllAuthors);
```

Comment:
For short lines like this one, you can easily put them on one line:
```suggestion
authorRouter.route('/').post(createAuthor).get(getAllAuthors);
```